### ISSUE TITLE
Clean up/refactor app.rb

### ISSUE THREAD

**Issue by neilparikh** [neilparikh/ScheduShare#1]
@MinghaoJ

If you're interested, we can spend an hour or so doing this at UofT hacks. You'd be able to learn some ruby along the way, and the code would be much easier to work with (if we decide to take it further)

**Comment by MinghaoJ** [comment]
Yes, after learning a whole lot more I think we can reimagine this project entirely.
